clients/drcov/CMakeLists.txt

```diff
@@ -175,20 +175,15 @@ get_property(dox_targets GLOBAL PROPERTY DynamoRIO_dox_targets)
 set_property(GLOBAL PROPERTY DynamoRIO_dox_targets ${dox_targets} ${doctgt})
```

Comment:
> i#3544 RV64: Open drcov test for RV64

nit: I find "Open" confusing: not sure what it means here.  "Enable" seems much clearer.

Reply:
Ok.